float32 heading
float32 speed
float32 heading_demand
float32 sway
float32 error
float32 int_error
float32 der_error
float32 aft_deadzone
float32 fwd_deadzone
float32 min_int_error
float32 max_int_error
float32 CS_Pgain
float32 CS_min 
float32 CS_max 
float32 CS_Pterm
float32 CSt
float32 CSb
float32 Thrust_Pgain
float32 Thrust_Igain
float32 Thrust_Dgain
float32 Thrust_Pmin
float32 Thrust_Pmax
float32 Thrust_Imin
float32 Thrust_Imax
float32 Thrust_Dmin
float32 Thrust_Dmax
float32 Thrust_Smin
float32 Thrust_Smax  
float32 Thrust_Pterm
float32 Thrust_Iterm
float32 Thrust_Dterm
float32 Thrust_heading
float32 Thrust_sway_min
float32 Thrust_sway_max
int32 thruster0
int32 thruster1
